routes/admin_fees.go

```diff
@@ -0,0 +1,210 @@
+package routes
+
+import (
+	"bytes"
+	"encoding/gob"
+	"encoding/json"
+	"fmt"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/deso-protocol/core/lib"
+	"github.com/golang/glog"
+	"io"
+	"net/http"
+)
+
+type TransactionFee struct {
```

Comment:
Let's add some comments to the struct fields here. In particular:
1) PublicKeyBase58Check --> Specify that this is the fee recipient.  At first I thought these fees were only applicable to specific public keys.
2) ProfileEntryResponse --> It looks like the ProfileEntryResponse is just used when the TransactionFee is part of an endpoint's Response but is typically nil when the frontend provides TransactionFees as part of a Request.  Would be helpful to clarify so that someone who is interacting with this API doesn't feel like they need to go dig up a ProfileEntryResponse in order to have it work properly.